lib/streamlit/__init__.py

```diff
@@ -204,7 +204,6 @@ def _update_logger() -> None:
 multiselect = _main.multiselect
 number_input = _main.number_input
 page_link = _main.page_link
-pills = _main.pills
```

Comment:
Removing it here so that the command is not exposed. I keep the tests etc.

Reply:
nit: with an underscore, you could even keep it in her `_pills = _main._pills` to apply almost no changes to all tests.

Reply:
Yeah thought about it but then decided against it to hide it even more, but I didn't have the container use case in mind. So next time 🙂 since the changes are done now, let's move on